Detections/SecurityEvent/TimeSeriesAnomaly-ProcessExecutions.yaml

```diff
@@ -64,4 +64,4 @@ entityMappings:
     fieldMappings:
       - identifier: FullName
         columnName: HostCustomEntity
-version: 1.0.0
\ No newline at end of file
+version: 1.0.0
```

Comment:
please update the version to 1.0.1. rest looks good. Thanks for PR

Reply:
Updated the version to 1.0.1. Thanks.